src/modules/75-cf/components/CreateFlagDialog/FlagDialog.tsx

```diff
@@ -16,19 +16,22 @@ import { ExitModalActions, Category, FeatureActions } from '@common/constants/Tr
 import useJira from '@cf/hooks/useJira'
 import { useFeatureFlag } from '@common/hooks/useFeatureFlag'
 import { FeatureFlag } from '@common/featureFlags'
+import { Tag } from 'services/cf'
 import { FlagTypeVariations } from './FlagDialogUtils'
 import FlagWizard from '../CreateFlagWizard/FlagWizard'
 import FlagTypeElement from '../CreateFlagType/FlagTypeElement'
 import CreateFlagButton from '../CreateFlagButton/CreateFlagButton'
 import css from './FlagDialog.module.scss'
 
-export interface FlagModalProps {
+export interface FlagDialogProps {
   disabled?: boolean
   environment: string
   isLinkVariation?: boolean
+  tags: Tag[]
+  tagsError: unknown
 }
 
-const FlagModal: React.FC<FlagModalProps> = ({ disabled, environment, isLinkVariation }) => {
+const FlagDialog: FC<FlagDialogProps> = ({ disabled, environment, isLinkVariation, tags, tagsError }) => {
```

Comment:
Might be worth defaulting these new things to be backwards compatible.

```suggestion
  tags?: Tag[]
  tagsError?: unknown
}

const FlagDialog: FC<FlagDialogProps> = ({ disabled, environment, isLinkVariation, tags = [], tagsError }) => {
```